dockerfiles/centos/streaming-integrator/docker-entrypoint.sh

```diff
@@ -32,5 +32,6 @@ test -d ${config_volume} && [[ "$(ls -A ${config_volume})" ]] && cp -RL ${config
 # copy any artifact changes mounted to artifact_volume
 test -d ${artifact_volume} && [[ "$(ls -A ${artifact_volume})" ]] && cp -RL ${artifact_volume}/* ${WSO2_SERVER_HOME}/
```

Comment:
@Aaquiff is mounting non-configuration, artifacts a valid use case in the case of new generation Enterprise Integrator profiles? Please verify this.

If not, we can remove this mount check.

Reply:
Mounting artifacts is still supported.